documentation/modules/rn-2.5.adoc

```diff
@@ -127,14 +127,19 @@ When migrating VMs that are installed with RHEL 9 as guest operating system from
 
 When adding an OVA provider, the error message `ConnectionTestFailed` may instantly appear, although the provider is created successfully. If the message does not disappear after a few minutes and the provider status does not move to `Ready`, this means that the `ova server pod creation` has failed. link:https://issues.redhat.com/browse/MTV-671[(MTV-671)]
 
-For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/browse/MTV-740?filter=12424645[Known Issues] in Jira.
+For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12424645[Known Issues] in Jira.
 
 
 [id="resolved-issues-25_{context}"]
 == Resolved issues
 
 This release has the following resolved issues:
 
+.Flaw was found in jsrsasign package which is vulnerable to Observable Discrepancy
+
+Versions of the package `jsrsasign` before 11.0.0, used in previous releases of {project-short}, are vulnerable to Observable Discrepancy in the RSA PKCS1.5 or RSA-OAEP decryption process. This discrepancy means an attacker could decrypt ciphertexts by exploiting this vulnerability. However, exploiting this vulnerability requires the attacker to have access to a large number of ciphertexts encrypted with the same key. One proposed workaround is to find and replace RSA and RSA-OAEP decryption with another crypto library. This issue has been resolved in {project-short} 2.5.5.
+
+For more information, see link:https://access.redhat.com/security/cve/CVE-2023-26159[CVE-2023-26159].
```

Comment:
this link is incorrect, should be https://access.redhat.com/security/cve/CVE-2024-21484